docker/mysql/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM bitnami/mysql:8.0.32-debian-11-r14
+
+# Install kopia to take backups
+COPY --from=kopia/kopia:latest /usr/bin/kopia /usr/local/bin/kopia
+
```

Comment:
Do we always want to use latest kopia binary? Or should we use one that kanister-tools image is using?

Reply:
I don't have much input on this. @viveksinghggits @ankitjain235 what should we do in this case ?

Reply:
Done !